cli/args/flags.rs

```diff
@@ -2265,6 +2287,31 @@ Specifying the filename '-' to read the file from stdin.
     )
 }
 
+fn serve_subcommand() -> Command {
+  runtime_args(Command::new("serve"), true, true)
+    .arg(
+      Arg::new("port")
+        .long("port")
+        .help("The TCP port to serve on, defaulting to 8000.")
+        .value_parser(value_parser!(NonZeroU16)),
+    )
```

Comment:
I don't think we should allow setting the host/port in the fetch configuration because those are always set by the "ambient environment" (ie: an embedder or webhoster).

Reply:
That's fair